### PR TITLE
Fix Done

### PR DESCRIPTION
**Pull request by dicolajuan**

### CODE REVIEW

File: frontend/src/layout/layout.component.tsx [hunk 1]
```diff
@@ -2,11 +2,17 @@ import React from 'react';
 
 import { NavBar } from './navbar';
 
-export const Layout: React.FC<{}> = ({ children }) => {
+interface Props {
```
Comment (d4v1d41) [inline frontend/src/layout/layout.component.tsx]: Let's move this into the interfaces folder. Try to follow the guidelines we are already using in that folder.

File: frontend/src/pages/dashboard.page.tsx [hunk 2]
```diff
@@ -1,5 +1,16 @@
 import React, { FC } from "react";
-
 import { RouteComponentProps } from "@reach/router";
+// import styles from '../styles/Dashboard.module.css';
```
Comment (d4v1d41) [inline frontend/src/pages/dashboard.page.tsx]: Let's remove the comments if they are not indeed needed.

File: frontend/src/routes.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ export function Routes() {
   return (
     <Layout>
       <Router>
-        <DashboardPage/>
+        <DashboardPage path="/dashboard" />
```
Comment (d4v1d41) [inline frontend/src/routes.tsx]: Let's add the default prop to this route:

image

Please, take a look at the documentation for Reach Router: https://reach.tech/router/tutorial/08-default-routes
For

File: frontend/src/layout/layout.component.tsx (hunk 1 above)
Comment (dicolajuan) [reply frontend/src/layout/layout.component.tsx]: I´ve moved the interface to a file inside the interfaces folder.

**Pull request merged**